src/Apps/Legion/Models/AppTypes.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * This file is for containing the top level types for use across the entire app. It should not contain any types specific to particular components or segments of the app.
```

Comment:
is there a reason why this is called "AppTypes" but not just "Types"?

Reply:
Yeah. I wanted a file just for the data types we use for the wizard. Maybe I should have called it Wizard.types.ts.